src/components/Home.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
 
-const Home = () => {
+import React from "react";
+import { Button } from "react-bootstrap";
+
+
```

Comment:
There should not be two consecutive newlines in a file

Reply:
I will fix it.